examples/ConvertPyImageJDataset.ipynb

```diff
@@ -0,0 +1,1153 @@
+      ],
+      "text/plain": [
+       "<xarray.DataArray (y: 408, x: 406)>\n",
+       "array([[160, 185, 208, ..., 192, 184, 166],\n",
+       "       [199, 215, 228, ..., 204, 196, 180],\n",
+       "       [226, 232, 236, ..., 212, 206, 193],\n",
+       "       ...,\n",
+       "       [203, 212, 216, ..., 167, 147, 143],\n",
+       "       [185, 198, 209, ..., 179, 163, 151],\n",
+       "       [156, 172, 189, ..., 191, 180, 164]], dtype=uint8)\n",
+       "Coordinates:\n",
+       "  * y        (y) float64 0.0 1.0 2.0 3.0 4.0 ... 403.0 404.0 405.0 406.0 407.0\n",
+       "  * x        (x) float64 0.0 1.0 2.0 3.0 4.0 ... 401.0 402.0 403.0 404.0 405.0"
+      ]
+     },
+     "execution_count": 8,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Clean attrs that are Java objects and cannot be serialized to Zarr\n",
+    "image_da.attrs.clear()\n",
```

Comment:
My goal with https://github.com/imagej/pyimagej/issues/200 would be to eliminate the need to do this. If a Java-side attribute can't be converted to something Pythonic, I think it's OK to drop it. We could at least make it configurable somehow.